source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/root.tsx

```diff
@@ -36,15 +37,17 @@ export default class RootPage extends React.Component {
         </section>
 
         <footer>
```

Comment:
this whole thing can be deleted

Reply:
ok, this is here because I created this branch a long time ago (last week ha!)

Reply:
:)

Reply:
deleted